base/BUILD

```diff
@@ -19,6 +19,7 @@ java_library(
         "//sdkcompat",
         "//third_party/auto_value",
         "@error_prone_annotations//jar",
+        "//third_party/scala",
```

Comment:
The 'base' module is used across all IDEs, and can't have a language specific dep (unless that language is supported in all JetBrains IDEs).

Reply:
fixed by moving to async messaging